src/components/route/addBook.jsx

```diff
@@ -22,15 +24,25 @@ const AddBook = () => {
   };
 
   return (
-    <div className="book-form">
-      <div>
-        <h3>ADD NEW BOOK</h3>
+    <div className="FormContianer-all">
+      <div className="FormContianer">
+        <div className="title">
+          <h3>ADD NEW BOOK</h3>
+        </div>
+
+        <form className="form">
+          <div className="input-container">
+            <input value={title} type="text" id="title" placeholder="Book Title Here..." onChange={(e) => setTitle(e.target.value)} />
+          </div>
+          <div className="input-container">
+            <input value={author} type="text" id="name" placeholder="Author Name Here..." onChange={(e) => setAuthor(e.target.value)} />
+          </div>
+          <div className="input-container">
+            <button type="button" value="AddNew" onClick={handleSubmit}> ADD BOOK </button>
+          </div>
+        </form>
```

Comment:
- [ ] Good job so far, I would advise you to add what is missing in your design(Logo icon and dropdown symbol). because your styling must be exactly the same as the one on Zeplin.
**Kindly pay attention to the red sign in the picture below to understand what I am talking about.**

- Your design:
![Capture1](https://github.com/misikir21/bookstore/assets/63972670/1bc02c6d-0a2b-4081-8b42-a880764f1624)

- Original Design
![Capture](https://github.com/misikir21/bookstore/assets/63972670/ff833b79-e13d-49fd-a035-b941c0c0793b)